apps/wallet-mobile/src/features/Settings/SettingsScreenNavigator.tsx

```diff
@@ -112,8 +112,8 @@ export const SettingsScreenNavigator = () => {
       <Stack.Screen //
         name="change-custom-pin"
         options={{
+          ...DEPRECATED_defaultStackNavigationOptions,
```

Comment:
pin screen is not revamped yet

![Simulator Screenshot - iPhone 14 Pro Max - 2023-08-11 at 11 43 34](https://github.com/Emurgo/yoroi/assets/105349292/3e14522b-5ab2-4d30-a7d2-b39115f969fe)